.travis.yml

```
@@ -11,8 +11,20 @@ matrix:
     env: NXF_VER='0.32.0' SUITE=test FLAGS=
     language: java
     jdk: openjdk8
-  - name: "Latest Nextflow version, regular test suite"
+  - name: "Latest Nextflow version, regular test suite (test --sketch_size_log2")"
```

```suggestion
  - name: "Latest Nextflow version, regular test suite"
```

it actually doesn't have those flags!